### ISSUE TITLE
@nrwl/cypress:migrate-to-cypress-10 on Windows ; will not rename spec files

### ISSUE THREAD

**Issue by jimhess** [nrwl/nx#12783]
## Current Behavior
Running `yarn nx @nrwl/cypress:migrate-to-cypress-10` to migrate to cypress 10. 
On Windows, some steps in the migration are missed or skipped.  Output does not indicate which.

## Expected Behavior
All upgrade steps completed
Compare to same steps executed on linux 

We expect all platforms to behave the same function.  When we had nrwl resources embedded in our teams, we noticed a similar hurdle to get the nrwl team to support windows platform.

## Steps to Reproduce
Run the command on Windows platform
Run the command on a linux or mac *(Ubuntu 20.x on wsl2 in my instance)*
Compare output or resulting file changes under git or version control tool.

Reproduced 3x on various machines / environments.

### Failure Logs
Ubuntu / linux log / output  (attached)
Windows log / output
 Note differences in executed commands ; windows section is missing commands that should have executed.

### Environment
```   Node : 16.13.0
   OS   : win32 x64
   yarn : 1.22.17
   nx : 14.8.0
   @nrwl/angular : 14.8.0
   @nrwl/cypress : 14.8.0
   @nrwl/detox : Not Found
   @nrwl/devkit : 14.8.0
   @nrwl/eslint-plugin-nx : 14.8.0
   @nrwl/expo : Not Found
   @nrwl/express : Not Found
   @nrwl/jest : 14.8.0
   @nrwl/js : 14.8.0
   @nrwl/linter : 14.8.0
   @nrwl/nest : Not Found
   @nrwl/next : Not Found
   @nrwl/node : Not Found
   @nrwl/nx-cloud : 15.0.0
   @nrwl/nx-plugin : Not Found
   @nrwl/react : Not Found
   @nrwl/react-native : Not Found
   @nrwl/schematics : Not Found
   @nrwl/storybook : 14.8.0
   @nrwl/web : Not Found
   @nrwl/workspace : 14.8.0
   typescript : 4.8.4
   ---------------------------------------
   Local workspace plugins:
   ---------------------------------------
   Community plugins:
         @ngneat/spectator: 10.0.1
```

[nwwl cypress migrate-to-cypress-10.txt](https://github.com/nrwl/nx/files/9852299/nwwl.cypress.migrate-to-cypress-10.txt)

**All files and names are lowercase - as to avoid case-sensitive file systems issues.**

**Comment by barbados-clemens** [comment]
this more than likely is due to the the paths not being normalized so when reading from the `@nrwl/devkit#Tree` all the paths 'did not exist' since the tree is unix based. Should be fixed in #12036 

Though I haven't seen this reported before and have had people have the migration run successfully on windows minus the imports not getting updated.

Do you have a cloneable project I can look at. there are also other 'edge' cases around when a cypress config is shared across projects.

**Comment by jimhess** [comment]
Sorry Caleb, no cloneable project or sample is available.
